src/legacy/core_plugins/console/public/np_ready/types/core_editor.ts

```diff
@@ -120,6 +136,11 @@ export interface CoreEditor {
    */
   clearSelection(): void;
 
+  /**
+   * TODO: Document
```

Comment:
Is this TODO supposed to be completed in this PR? The other new functions have documentation.